docs/high-availability/etcd.manifest

```diff
@@ -0,0 +1,104 @@
+{
+"apiVersion": "v1beta3",
```

Comment:
I think that's okay, we can update to v1 and merge this ;Then changes made will be valuable to people .